master/buildbot/test/unit/test_status_buildstep.py

```diff
@@ -13,15 +13,48 @@
 #
 # Copyright Buildbot Team Members
 
+import datetime
 import os
 from twisted.trial import unittest
+from twisted.internet import defer
+from buildbot.db import buildrequests, builds
 from buildbot.status import builder, master
-from buildbot.test.fake import fakemaster
+from buildbot.test.fake import fakemaster, fakedb
+from buildbot.test.util import connector_component
+from buildbot.util import UTC, epoch2datetime
 
-class TestBuildStepStatus(unittest.TestCase):
+class TestBuildStepStatus(connector_component.ConnectorComponentMixin,
```

Comment:
This tests shouldn't use a connection to db. We can mock database function because it is tested in separate unit test

Reply:
I changed fakedb to StubDB and StubBuildRequest.